app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/UndertakingController.scala

```diff
@@ -16,30 +16,40 @@
 
 package uk.gov.hmrc.eusubsidycompliancefrontend.controllers
 
+import cats.implicits.{catsSyntaxOptionId}
+import play.api.Configuration
+
 import javax.inject.{Inject, Singleton}
 import play.api.data.Form
 import play.api.data.Forms.mapping
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.eusubsidycompliancefrontend.actions.EscActionBuilders
+import uk.gov.hmrc.eusubsidycompliancefrontend.actions.requests.EscAuthRequest
 import uk.gov.hmrc.eusubsidycompliancefrontend.config.AppConfig
-import uk.gov.hmrc.eusubsidycompliancefrontend.models.{BusinessEntity, Undertaking}
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.Language.{English, Welsh}
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.emailSend.EmailParameters.SingleEORIEmailParameter
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.{BusinessEntity, Language, Undertaking}
 import uk.gov.hmrc.eusubsidycompliancefrontend.models.types.{EORI, Sector, UndertakingName, UndertakingRef}
-import uk.gov.hmrc.eusubsidycompliancefrontend.services.{EscService, Store, UndertakingJourney}
+import uk.gov.hmrc.eusubsidycompliancefrontend.services.{EscService, RetrieveEmailService, SendEmailService, Store, UndertakingJourney}
 import uk.gov.hmrc.eusubsidycompliancefrontend.views.html._
 
+import java.util.Locale
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class UndertakingController @Inject()(
-  mcc: MessagesControllerComponents,
```

Comment:
We should agree on a coding style and stick to it otherwise we'll get a lot of noise in the diffs like this. I prefer the smaller indent but if the standard here is different then we should align to that.

Reply:
yeah here scalaformat file is missing i guess :) Even this is new to me and there are lot of formatting changes which needs to be done